canonicalwebteam/discourse/parsers/engageParser.py

```diff
@@ -124,6 +124,7 @@ def parse_topic(self, topic):
             "updated": humanize.naturaltime(
                 updated_datetime.replace(tzinfo=None)
             ),
+            "related": self.current_topic_related,
```

Comment:
Is there a reason we are setting this attribute on self? could we not just call `self._parse_related` here?

Reply:
No. Removed